exercises/book-store/src/example/java/BookStore.java

```diff
@@ -0,0 +1,76 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BookStore{
```

Comment:
In English we would say `Bookstore` rather than `BookStore`.  Also, the folder name should likewise be one word `bookstore` without the hypen.